CMakeLists.txt

```diff
@@ -21,8 +23,6 @@ if(MEMORY_POOL_DEBUG)
   target_compile_definitions(mem_pool PUBLIC -DCONFIG_MEMORY_POOL_DEBUG=1)
 endif()
 
-set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
Because it won't take effect here, it must be prepared before starting the build